erpnext_com/www/docs/user/manual/en/using-erpnext/articles/keyboard-shortcuts.md

```diff
@@ -0,0 +1,38 @@
+<!-- add-breadcrumbs -->
+#Keyboard Shortcuts
+
+ERPNext Version 12 is almost fully accessible using keyboard shortcuts.
```

Comment:
```suggestion
ERPNext Version 12 is almost fully accessible using keyboard shortcuts. The shortcuts are different for macOS, Windows, and Linux. The key combination will depend on the OS you are using.
```
verify if correct

Reply:
I don't think this needs to be mentioned since the only difference is the command key and ctrl key and that is specified on the dialog.

Reply:
@prasadarr It's not necessary to mention this. Ctrl key works the same on Windows and Linux, and the MacOS command key is the meta key, is understood by all Mac users.

Reply:
@scmmishra I got it the first time when @prssanna mentioned it. Thanks for explaining it anyway. If you have a lot of free time try leaving some productive review comments.